src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -121,7 +121,7 @@
     <value>PowerShell {0}
```

Comment:
@SteveL-MSFT, your last commit had 10 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

TabCompletion.Should complete abbreviated cmdlet
```powershell
Expected exactly 'Import-PowerShellDataFile', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 16
16:         $res | Should -BeExactly 'Import-PowerShellDataFile'
```
TabCompletion.Should complete abbreviated function
```powershell
Expected the actual value to be greater than or equal to 1, but got 0.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 21
21:         $res.Count | Should -BeGreaterOrEqual 1
```
Temp: drive.TEMP: drive exists
```powershell
Expected exactly 'Temp', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-PSDrive.Tests.ps1: line 64
64:         $res.Name | Should -BeExactly "Temp"
```
Get-Command Feature tests.-UseAbbreviationExpansion tests.Can return multiple results relying on auto module loading
```powershell
Expected a collection with size 2, but got an empty collection.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1: line 39
39:             $results | Should -HaveCount 2
```
Get-Command Feature tests.-UseAbbreviationExpansion tests.Can return multiple results for cmdlets matching abbreviation
```powershell
Expected 'Invoke-Command' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1: line 69
69:             $results.Name | Should -Contain "Invoke-Command"
```